homie/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "homie"
+version = "0.1.0"
+authors = ["Andrew Walbran <qwandor@google.com>", "David Laban <alsuren@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+async-channel = "1.4.0"
+futures = "*"
+local_ipaddress = "0.1"
+log = "*"
+mac_address = "1.0"
+rumqttc = { git = "https://github.com/bytebeamio/rumqtt" }
```

Comment:
have they made a release with your changes yet?

Reply:
Looks like it. Moved to that in #19